charts/stable/channels-dvr/Chart.yaml

```diff
@@ -17,7 +17,7 @@ maintainers:
     url: https://truecharts.org
 name: channels-dvr
 sources:
-  - https://github.com/truecharts/charts/tree/master/charts/incubator/channels-dvr
+  - https://github.com/truecharts/charts/tree/master/charts/stable/channels-dvr
 type: application
 version: 1.0.5
```

Comment:
version should be bumped.
i typically have done major bumps but cant recall if thats the case anymore.

Reply:
@xstar97 Version bumped from 1.0.5 to 2.0.0. Thank you.